docs/content/deprecation/features.md

```diff
@@ -4,31 +4,10 @@ This page is maintained and updated periodically to reflect our roadmap and any
 
 | Feature                                                     | Deprecated | End of Support | Removal |
 |-------------------------------------------------------------|------------|----------------|---------|
-| [Pilot](#pilot)                                             | 2.7        | 2.8            | 2.9     |
-| [Consul Enterprise Namespace](#consul-enterprise-namespace) | 2.8        | N/A            | 3.0     |
 | [TLS 1.0 and 1.1 Support](#tls-10-and-11)                   | N/A        | 2.8            | N/A     |
```

Comment:
Why not removing the notion of TLS in this PR? Traefik v3 users should not have to worry about this breaking change too.

Reply:
Maybe we want drop the whole table.